instrumentation/jms-1.1/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/jms/JmsMessageConsumerInstrumentation.java

```diff
@@ -57,21 +55,15 @@ public static long onEnter() {
 
     @Advice.OnMethodExit(onThrowable = Throwable.class, suppress = Throwable.class)
     public static void stopSpan(
-        @Advice.This MessageConsumer consumer,
         @Advice.Enter long startTime,
         @Advice.Return Message message,
         @Advice.Thrown Throwable throwable) {
       MessageDestination destination;
       if (message == null) {
-        destination =
```

Comment:
`JmsSessionInstrumentation` can be removed now, right?

Reply:
removed